sdks/java/io/clickhouse/src/main/java/org/apache/beam/sdk/io/clickhouse/ClickHouseIO.java

```diff
@@ -111,7 +111,9 @@
  * <tr><td>{@link TableSchema.TypeName#BOOL}</td> <td>{@link Schema.TypeName#BOOLEAN}</td></tr>
  * </table>
  *
- * Nullable row columns are supported through Nullable type in ClickHouse.
+ * Nullable row columns are supported through Nullable type in ClickHouse. Adding support for <a
```

Comment:
This is javadoc. The statement "Adding support ..." sounds like an announcement and consider moving it to [CHANGES.md ](https://github.com/apache/beam/blob/master/CHANGES.md)

For here, consider phrasing like "Low cardinality hint is supported through LowCardinality DataType in ClickHouse."